contracts/interchain-token-service/src/interface.rs

```diff
@@ -49,6 +51,14 @@ pub trait InterchainTokenServiceInterface: AxelarExecutableInterface {
         gas_token: Token,
     ) -> Result<BytesN<32>, ContractError>;
 
+    fn deploy_remote_canonical_token(
```

Comment:
Docstring should be added on the interface/trait. See rust doc convention for trait vs impl documentation